source-manager/src/main/java/com/jointrivial/sourcemanager/nordigen/repository/NordigenConnectionIdRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.jointrivial.sourcemanager.nordigen.repository;
+
+import com.jointrivial.sourcemanager.nordigen.model.entity.NordigenConnectionId;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface NordigenConnectionIdRepository extends JpaRepository<NordigenConnectionId, String> {
+
+    NordigenConnectionId getRequisitionIdByReferenceId(String referenceId);
```

Comment:
Here, you don't need the whole object. Just the requisitionId, which is a String. Also add getBankNameByReferenceId for later usage.

Reply:
No, I use the whole object and if I split the request into two separate ones, this will lead to 2 calls to the repository, now I make only one.

Reply:
Alright, I must agree that this makes a lot of sense in a bigger repo. However, ours is not likely to become any bigger, but yeah, we can leave it this way.